src/components/checkbox/checkbox.js

```diff
@@ -150,7 +150,7 @@ function MdCheckboxDirective(inputDirective, $mdAria, $mdConstant, $mdTheming, $
         }
       }
       function listener(ev) {
-        if (element[0].hasAttribute('disabled')) {
+        if (element[0].hasAttribute('disabled') || scope.skipToggle) {
```

Comment:
@EladBezalel Should I add a comment here to explain what `skipToggle` is used for?